oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java

```diff
@@ -268,7 +268,8 @@ public Builder setAwsSecurityCredentialsSupplier(
      * Sets the AWS regional credential verification URL. If set, will override any credential
      * verification URL provided in the credential source. If not set, the credential verification
      * URL will default to
-     * https://sts.{region}.amazonaws.com?Action=GetCallerIdentity&Version=2011-06-15"
+     *
+     * <p><a href=https://sts.{region}.amazonaws.com?Action=GetCallerIdentity&Version=2011-06-15>
```

Comment:
Should <p> and <a href> have a closing tag as well?

Reply:
Was not intentionally using <p>, let me try to remove it.

Reply:
Hmmmm. lint is doesn't allow me to remove it and failed. Adding a closing tag also failed it.